chase/frontend/app/login/participant/page.tsx

```diff
@@ -0,0 +1,67 @@
+"use client";
+import React, { useState, useContext } from "react";
+import Image from "next/image";
+
+import UsernameLogin from "../../../components/login/username";
```

Comment:
We should look up how to configure path alias for imports. This way we could reduce
`../../../components/login/username`
down to
`$components/login/username`
Relative imports get messy as the application grows

Reply:
The setup process requires some changes in various config files. 
Have a look here: https://nextjs.org/docs/pages/building-your-application/configuring/absolute-imports-and-module-aliases